packages/fxa-auth-server/test/local/routes/subscriptions.js

```diff
@@ -468,6 +470,42 @@ describe('subscriptions', () => {
     });
   });
 
+  describe('PUT /oauth/subscriptions/active/{subscriptionId}', () => {
+    it('should allow updating of subscription plan', async () => {
+      subhub.stripeHelper.verifyPlanUpgradeForSubscription = sinon.fake();
+      subhub.stripeHelper.changeSubscriptionPlan = sinon.fake();
+      // TODO: TBD subhub response for updatePayment, do something with it?
```

Comment:
nit: Looks like copypasta from the updatePayment test - don't think we need a TODO here